src/pages/ApartmentShow.js

```diff
@@ -31,6 +31,9 @@ const ApartmentShow = ({ apartments }) => {
               <NavLink to={`/apartmentindex`} className="nav-link">
                 <Button className="apartment-button">Back to Listings</Button>
               </NavLink>
+              <NavLink to={"/apartmentedit/:id"} className="nav-link">
```

Comment:
Does this route work if you click on an apartment? How do you know which apartment to navigate to?